unity/src/main/java/io/delta/unity/UCCatalogManagedClient.java

```diff
@@ -198,6 +304,30 @@ private String getVersionOrTimestampString(
     }
   }
 
+  private String getCommitRangeBoundariesString(
+      Optional<Long> startVersionOpt,
+      Optional<Long> startTimestampOpt,
+      Optional<Long> endVersionOpt,
+      Optional<Long> endTimestampOpt) {
+    String startBound;
+    if (startVersionOpt.isPresent()) {
+      startBound = startVersionOpt.get() + "(version)";
```

Comment:
nit: space before the bracket? within the string? up to you

Reply:
I thought about it, I thought this was the most clear since we format the "=" without a space (here and for the snapshot load string)